data/fabian4.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "fabian4",
+    "type": "personal",
+    "bio": "A backend developer with the knowledge of BigData / frontend / full-stack that is able and willing to do ANYTHING 🚀",
+    "socials": [
+        {
+            "icon": "FaTwitter",
+            "url": "https://twitter.com/fabian13416618"
+        },
+        {
+            "icon": "FaInstagram",
+            "url": "https://www.instagram.com/fabian__bao"
+        },
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/fabian4"
+        },
+        {
+            "icon": "FaLinkedin",
+            "url": "https://linkedin.com/in/fabian4"
```

Comment:
Your LinkedIn profile link doesn't exist, please add a valid URL

![image](https://user-images.githubusercontent.com/88102392/224943070-11e3d695-c07c-4063-80f6-b30f1d00270f.png)

Reply:
You can reach it now